src/BenchmarkDotNet/Configs/DefaultConfig.cs

```diff
@@ -37,7 +37,10 @@ public IEnumerable<IExporter> GetExporters()
 
         public IEnumerable<ILogger> GetLoggers()
         {
-            yield return ConsoleLogger.Default;
+            if (AppDomain.CurrentDomain.FriendlyName.StartsWith("LINQPad Query Server", StringComparison.OrdinalIgnoreCase))
```

Comment:
It would be better to have all LINQ-specific logic in a single class. I suggest hiding this check inside LinqPadLogger. We can introduce a public static field `LinqPadLogger.IsAvaliable`.

Reply:
Great suggestion; fixed.